sql/core/src/test/resources/sql-tests/results/describe-table-column.sql.out

```diff
@@ -0,0 +1,133 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 15
+
+
+-- !query 0
+CREATE TEMPORARY VIEW desc_col_temp_table (key int COMMENT 'column_comment') USING PARQUET
+-- !query 0 schema
+struct<>
+-- !query 0 output
+
+
+
+-- !query 1
+DESC desc_col_temp_table key
+-- !query 1 schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query 1 output
+col_name 	data_type 	comment        	
+key      	int       	column_comment
+
+
+-- !query 2
+DESC EXTENDED desc_col_temp_table key
+-- !query 2 schema
+struct<col_name:string,data_type:string,min:string,max:string,num_nulls:string,distinct_count:string,avg_col_len:string,max_col_len:string,comment:string>
+-- !query 2 output
+col_name 	data_type 	min  	max  	num_nulls 	distinct_count 	avg_col_len 	max_col_len 	comment        	
```

Comment:
can you check with hive? I feel this output is not friendly to users. I'd like to see something like:
schema: <info: string, value: string>
output:
```
col_name   abc
data_type  int
max           3
....
```